pkgs/development/tools/analysis/codeql/default.nix

```diff
@@ -27,6 +27,7 @@ stdenv.mkDerivation rec {
     jdk11
     stdenv.cc.cc.lib
     curl
+    lttng-ust
```

Comment:
I'm not sure how I feel about adding a speculative dependency when it's not strictly necessary for today's build. Increases the closure for no particular reason. Seems more prudent to just add it when the upstream maintainers change their build process. Also, is there a reference upstream issue/patch to link to?

Reply:
Fair callout. I'm in convo with the maintainers, will share back a public issue when it is available, will remove the extra dependency for now.